automl/noxfile.py

```diff
@@ -138,3 +139,22 @@ def cover(session):
     session.run("coverage", "report", "--show-missing", "--fail-under=83")
 
     session.run("coverage", "erase")
+
+@nox.session(python="3.7")
+def docs(session):
+    """Build the docs for this library."""
+
+    session.install('-e', '.')
+    session.install('sphinx', 'alabaster', 'recommonmark')
```

Comment:
Docs build [fails](https://source.cloud.google.com/results/invocations/2c812072-8095-4de2-aec5-b0b1f874dfc5/targets/cloud-devrel%2Fclient-libraries%2Fgoogle-cloud-python%2Fpresubmit%2Fautoml/log) because 'sphinx_rtd_theme' is still configured.  See #8021.